### PR TITLE
rio: 0.1.10 -> 0.1.13

### PR DESCRIPTION
**Pull request by otavio**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by TornaxO7** [comment]
@otavio you could also update this directly to `0.1.12`

**Comment by otavio** [comment]
@TornaxO7 I have sent it.

**Comment by TornaxO7** [comment]
peepoAwesome

**Review by TornaxO7 — APPROVED** [review]

**Review by TornaxO7 — APPROVED** [review]

File: pkgs/applications/terminal-emulators/rio/default.nix [hunk 1]
```diff
@@ -123,7 +123,7 @@ rustPlatform.buildRustPackage rec {
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ tornax otavio oluceps ];
     platforms = lib.platforms.unix;
-    changelog = "https://github.com/raphamorim/rio/blob/v${version}/CHANGELOG.md";
+    changelog = "https://github.com/raphamorim/rio/blob/v${version}/docs/releases.md";
```
Comment (TornaxO7) [inline pkgs/applications/terminal-emulators/rio/default.nix:126]: Wait, shouldn't that be https://github.com/raphamorim/rio/blob/main/docs/docs/releases.md ?

**Review by TornaxO7 — CHANGES_REQUESTED** [review]

File: pkgs/applications/terminal-emulators/rio/default.nix (hunk 1 above)
Comment (TornaxO7) [reply pkgs/applications/terminal-emulators/rio/default.nix:126]: I can't open up https://github.com/raphamorim/rio/blob/v0.1.10/docs/releases.md for example

**Review by TornaxO7 — APPROVED** [review]
Nice! Changelog link works now!

**Comment by bbigras** [comment]
Result of `nixpkgs-review pr 340344` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>:white_check_mark: 2 packages built:</summary>
  <ul>
    <li>rio</li>
    <li>rio.terminfo</li>
  </ul>
</details>

**Review by bbigras — APPROVED** [review]

**Comment by adamcstephens** [comment]
The aarch64-linux build failed.

**Comment by otavio** [comment]
> The aarch64-linux build failed.

@adamcstephens  Is it something that you can help with?

**Comment by adamcstephens** [comment]
Probably not, I have no knowledge of this package.

**Comment by NickCao** [comment]
Please try bumping to 0.1.13, should be fixed by https://github.com/raphamorim/rio/commit/ff3afb7f28ee427590547cc260a7d78c5ae59140

**Comment by otavio** [comment]
I have just sent it.

**Review by NickCao — APPROVED** [review]

**Pull request merged**